docs/content/operator/common/network.mdx

```diff
@@ -0,0 +1,30 @@
+import TestnetNodeConfig from "../../_snippets/network-node-config-testnet.mdx";
+import DevnetNodeConfig from "../../_snippets/network-node-config-devnet.mdx";
+
+To synchronize the state, the node needs to establish connections with other nodes that have already joined the network. Ensure that port 8084 is open and set up an external address for node discovery.
+
+Edit the node configuration `.yaml` file to include peer nodes for state synchronization.
+Add the following lines to the node's configuration, based on the network you'd like to connect to:
+
+<Tabs groupId="network">
+    <TabItem label="Testnet" value="testnet">
+        <TestnetNodeConfig />
+    </TabItem>
+    <TabItem label="Devnet" value="devnet">
+        <DevnetNodeConfig />
+    </TabItem>
+</Tabs>
+
+## Connectivity
+
+The IOTA node uses the following ports by default:
+
+| Protocol/Port | Reachability     | Purpose                           |
+| ------------- | ---------------- | --------------------------------- |
+| TCP/8080      | Inbound          | Validator/transaction interface   |
+| TCP/8081      | Inbound/Outbound | Consensus interface               |
+| UDP/8084      | Inbound/Outbound | Peer-to-peer state sync interface |
+| TCP/8443      | Outbound         | Metrics pushing                   |
+| TCP/9184      | Localhost        | Metrics scraping                  |
+
+To run a validator successfully, it is critical that ports 8080-8084 are open as outlined, including the specific protocol (TCP/UDP).
```

Comment:
And what about fullnodes?

Reply:
I removed this section from common/network, and add it to both validator and full node: 1ef5ddad62be450f817f9cbcb2266abfc03214b3